docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -4,5 +4,63 @@
 Vitamin A Deficiency
 ====================
 
+Disease Description
+-------------------
+
+Vitamin A deficiency (VAD) is a lack of vitamin A in blood and tissues.
+Vitamin A deficiency is considered as one of the most serious public health concerns in developing countries
+and can contribute directly or indirectly to disability. 
+
+GBD 2017 Modeling Strategy
+------------------------------------
+In Global Burden of Disease (GBD) 2017, VAD exposure definition is proportion of the population with serum retinol concentration <0·7 μmol/L.
+Like iron deficiency, the cause VAD is also a population attributable fraction (PAF) of 1 cause with the VAD risk factor. That is, 100% of the VAD cases are attributable 
+to the VAD risk factor. VAD Risk exposure and VAD cause prevalence data are the same.
+
+Vitamin A Deficiency Cause
++++++++++++++++++++++++++++++
+
+The VAD in GBD 2017 that is 100% attributable to the 
+VAD risk factor. The VAD cause in GBD is a 
+YLD-only cause, meaning that it contributes to morbidity, but not mortality.
```

Comment:
I would add a section about the cause and/or risk factor restrictions tables. This is wher eyou can add the above assumptions in. Maybe also add the limitations here, in the future.